.github/workflows/fetch-data.yaml

```diff
@@ -0,0 +1,87 @@
+on:
+  schedule:
+    - cron: "0 0 1,15,30 3-11 *"
+    
+  push:
+    branches:
+    - '14-automate-sk-dataflow'
+    - main
+
+name: fetch-data
+
+jobs:
+  fetch-data:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: ubuntu-latest,   r: 'release'}
+    env:
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      RSPM: ${{ matrix.config.rspm }}
+      GITHUB_TOKEN: ${{ secrets.AUTOMATISATION }}
+      GSHEET_KEY: ${{ secrets.GSHEET_KEY }}
+      GSHEET_SECRET: ${{ secrets.GSHEET_SECRET }}
+      GSHEET_ID: ${{ secrets.GSHEET_ID }}
+      ENV_TEST: ${{ secrets.ENV_TEST }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.AUTOMATISATION }}
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: ${{ matrix.config.r }}
+          http-user-agent: ${{ matrix.config.http-user-agent }}
+
+      - name: Setup pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+
+      - name: Install linux libraries
+        run: |
+          sudo apt install libsodium-dev
+          sudo apt-get install libcurl4-openssl-dev
+          sudo apt-get install libgdal-dev libproj-dev
+          sudo apt install libudunits2-dev
+
+      - name: Install R packages
+        run: |
+          source("script/install_packages.R")
+        shell: Rscript {0}
+        
+      - name: Fetch data
+        run: |
+          source("script/run_fetch_data.R")
+        shell: Rscript {0}
+
+      - name: Commit and push changes
+        uses: devops-infra/action-commit-push@master
+        with:
+          github_token: ${{ secrets.AUTOMATISATION }}
+          commit_prefix: "[AUTO]"
+          commit_message: "Fetch data"
+          target_branch: automatic-update
+          add_timestamp: true
+
```

Comment:
@DimEvil automate dwc mapping as well ? 

ifso, which do you prefer ?
1. add you as reviewer  - or -
2. `@` mention you in pull request - or -
3. other - or -
4. neither